scripts/debug_tools/prepare_analyzer_cmd.py

```diff
@@ -26,9 +26,18 @@ def __init__(self, opts):
 
     def __call__(self, path):
 
-        if re.search('clang$', path):
+        if re.search('clang(-(\d)+)?$', path):
```

Comment:
Could you please add a comment with an example for the matches: `clang-7` 
Would it work with minor versions too? Could we have `clang-7.0.1` ?

Reply:
I checked only in my build directory, and there is "clang-11" only. I think this is generated by default to differentiate between main releases.